### DISCUSSION TITLE
Keybind while in i3lock

### DISCUSSION THREAD

**Discussion by d-hain** [i3/i3#5725] [Q&A] [answered]
I want to have one keybind that always works, also while locked and while the lid of my laptop is closed.

I think that is all the necessary info.
If someone needs to know more just ask.

**Comment by okraits** [comment, marked as answer]
If that would be possible, it would be a security issue. i3lock consumes all key events and locks your screen, making it impossible to interact with your computer unless you enter the correct password.

**Reply by orestisfl** [reply]
For more context, here's i3locks issue: https://github.com/i3/i3lock/issues/52 and [i3lock-color](https://github.com/Raymo111/i3lock-color) seems to have an option to allow media keys to pass through.
